scripts/lighthouseRunner.js

```diff
@@ -0,0 +1,117 @@
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+
+const thresholdTypes = [
+  'acessibility',
+  'seo',
+  'best-practices',
+  'pwa',
+  'performance',
+];
+
+if (process.argv.length < 3) {
```

Comment:
Because the first two items in `process.argv` are not needed by the script, but the last 5 are the params which are

```
[ '/Users/myUsername/.nvm/versions/node/v8.11.3/bin/node',
  '/Users/myUsername/Sites/simorgh/scripts/lighthouseRunner.js',
  '--accessibility=2',
  '--seo=0',
  '--best-practices=0',
  '--pwa=0',
  '--performance=0' ]
```

To reproduce what I mean, do `console.log(process.argv);` and run the script, e.g. 
`node scripts/lighthouseRunner.js --accessibility=2 --seo=0 --best-practices=0 --pwa=0 --performance=0`

Reply:
If you wanted to be explicit in pulling out the params you could look at an example in the validator for `DEBUG_MODE`. It get's passed to the node process [here](https://github.com/BBC-News/simorgh/blob/latest/package.json#L15) and then gets used [here](https://github.com/BBC-News/simorgh/blob/latest/src/app/dataValidator/utilities/messaging.js#L2)

This would mean you could explicitly pull out each param like so:
```
const accessibilityThreshold = process.env.npm_config_accessibility;
```

Reply:
Hmmmm weirdly I can't actually get the above example working, which is strange because it works for the validator 🤷‍♂️

Reply:
Similar to my suggestion but slightly different would be to use env variables on the process. These are set inline prior to running the process - that would mean having:
```accessibility=2 seo=0 best-practices=0 pwa=0 performance=0 node scripts/lighthouseRunner.js```

Then to access that these env var's would be:
`const accessibilityThreshold = process.env.accessibility;`

Reply:
I think it would be cleaner and easier to use to pass all config in as a file, and just pass in the file path in as an environment variable or similar.

Reply:
👍